vignettes/BiocParallel_BatchtoolsParam.Rmd

```diff
@@ -0,0 +1,229 @@
+---
+title: "Introduction to *BatchtoolsParam*"
+author:
+- name: "Nitesh Turaga"
+  email: Nitesh.Turaga@RoswellPark.org
+- name: "Martin Morgan"
+  email: Martin.Morgan@RoswellPark.org
+date: "Edited: March 22, 2018; Compiled: `r format(Sys.time(), '%B %d, %Y')`"
+vignette: >
+  %\VignetteIndexEntry{2. Introduction to BatchtoolsParam}
+  %\VignetteKeywords{parallel, Infrastructure}
+  %\VignettePackage{BiocParallel}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document:
+    number_sections: true
+    toc: yes
+    toc_depth: 4
+---
+ 
+# Introduction
+
+The class is an interface to `batchtools` the package from within `BiocParallel`. This aims to
+replace BatchjobsParam as `BiocParallel`'s class for computing on a high performance
+cluster such as SGE, TORQUE, LSF, SLURM, OpenLava.
+
+# Quick start
+
+This example demonstrates the easiest way to launch a 100000 jobs using
+batchtools. The first step involves creating a `BatchtoolsParam` class. You can compute
+using 'bplapply' and then the result is stored.
+
+
+```{r intro}
+library(BiocParallel)
+
+## Pi approximation
+piApprox <- function(n) {
+    nums <- matrix(runif(2 * n), ncol = 2)
+    d <- sqrt(nums[, 1]^2 + nums[, 2]^2)
+    4 * mean(d <= 1)
+}
+
+piApprox(1000)
+
+## Apply piApprox over
+param <- BatchtoolsParam()
+result <- bplapply(rep(10e5, 10), piApprox, BPPARAM=param)
+mean(unlist(result))
+```
+
+# *BatchtoolsParam* interface
+
+The interface allows you to replace the BatchJobsParam interface, and
+allow more intuitive usage of your high performance cluster with `BiocParallel`.
```

Comment:
Slight inconsistency here. Please update this section to: 

The `BatchtoolsParam` interface allows you to replace the BatchJobsParam interface, and allow more intuitive usage of your high performance cluster with *[BiocParallel](http://bioconductor.org/packages/release/bioc/html/BiocParallel.html)*.